.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,3 +19,15 @@ the linked issue.
 -->
 
 Related to #
+
+#### Things to check
+
+Complete all of the items on this checklist **before** merging the PR into the trunk. Both the PR author and review are responsible for ensuring the checklist is completed. 
+
+Not all items will apply to every PR, and any inapplicable items can just be marked as complete.
```

Comment:
I like where this is going but think it could be more direct. Would also prefer if we left things unchecked instead of checking them off regardless - easier to convey what was done or not done at a glance.

```suggestion
#### Pre-merge Checklist

Complete applicable items on this checklist **before** merging into trunk. Inapplicable items can be left unchecked.

Both the PR author and reviewer are responsible for ensuring the checklist is completed.
```